docs/developing-prometheus-rules-and-grafana-dashboards.md

```diff
@@ -305,3 +305,23 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') + {
 { ['prometheus-' + name]: kp.prometheus[name] for name in std.objectFields(kp.prometheus) } +
 { ['grafana-' + name]: kp.grafana[name] for name in std.objectFields(kp.grafana) }
 ```
+
+Incase you have lots of json dashboard exported out from grafan UI the above approch is going to take lots of time. to improve performance we can use `rawGrafanaDashboards` field and provide it's value as json string by using importstr
+```jsonnet
```

Comment:
Can you put this in a file in the `examples/` directory and add the `[embedmd]` tag like the above snippet? Then we'll run this script on CI and make sure what's in the docs is actually working code :slightly_smiling_face: .

Reply:
updated.
it seems test is failing because of some other issue.